x-pack/plugins/rule_registry/server/search_strategy/search_strategy.ts

```diff
@@ -146,6 +146,9 @@ export const ruleRegistrySearchStrategyProvider = (
         map((response) => {
           // Do we have to loop over each hit? Yes.
           // ecs auditLogger requires that we log each alert independently
+          //
+          //
+          logger.debug(JSON.stringify({ RuleStrategy: response }));
```

Comment:
I think this code should be removed and was added for the testing purposes

Reply:
@YulNaumenko , you are right, removed.